<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1080</width>
      <height>531</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="manager">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>921</width>
        <height>421</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="showAllBtn">
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>60</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>showAll</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addBtn">
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>100</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>add</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteBtn">
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>140</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>delete</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>副本类型:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="typeCB">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>25</y>
        <width>100</width>
        <height>25</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>无</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>迷宫挑战</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>行会令</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>讨伐歼灭战</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>大型任务</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>绝境战</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>30</y>
        <width>40</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>职业:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="jobCB">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>25</y>
        <width>90</width>
        <height>25</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>无</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>骑士</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>战士</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>暗黑骑士</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>绝枪战士</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>武僧</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>龙骑士</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>忍者</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>武士</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>钐镰客</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>吟游诗人</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>机工士</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>舞者</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>黑魔法师</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>召唤师</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>赤魔法师</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>白魔法师</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>学者</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>占星术士</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>贤者</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>基础职业</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>30</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string> 副本名称:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nameLE">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>25</y>
        <width>191</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="searchBtn">
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>25</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>search</string>
      </property>
     </widget>
     <widget class="QPushButton" name="submitBtn">
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>180</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>submit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="revertBtn">
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>220</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>revert</string>
      </property>
     </widget>
     <widget class="QComboBox" name="versionCB">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>25</y>
        <width>100</width>
        <height>25</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>无</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>重生之境</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>苍穹之禁城</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>红莲之狂潮</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>暗影之逆焰</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>晓月之终途</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>30</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>副本版本:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="analysis">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
